packages/app/src/components/catalog/shared/InfoCard.tsx

```diff
@@ -127,21 +152,40 @@ export const InfoCard = () => {
     },
   ];
   return (
-    <Card title="Information" subheader={entity.metadata.description} noPadding>
-      <Table
-        options={{
-          search: false,
-          paging: false,
-          toolbar: false,
-          header: false,
-          padding: 'dense',
-        }}
-        data={data}
-        columns={[
-          { field: 'name', highlight: true, width: '30%', cellStyle: { whiteSpace: 'nowrap'} },
-          { field: 'value' },
-        ]}
+    <Card noPadding>
+      <CardHeader
+        title="Information"
+        avatar={
+          logoUrl &&
+          <Avatar className={classes.avatar}
+            variant="square"
+            alt={`${entity.metadata.name} logo`}
+            src={logoUrl}
+          />
+        }
+        titleTypographyProps={{ variant: 'h5' }}
       />
+
+      <CardContent className={classes.table} >
+        <Typography className={classes.subheader} >
+          {entity.metadata.description}
+        </Typography>
```

Comment:
Can you please move this into a `subheader`? Typographically it's cleaner if those 2 text vertically align :slightly_smiling_face:

Reply:
Changed in the lastest commit.